generated/nidigital/nidigital/unit_tests/test_nidigital.py

```diff
@@ -29,6 +29,12 @@ def setup_method(self, method):
         self.patched_library.niDigital_UnlockSession.side_effect = self.side_effects_helper.niDigital_UnlockSession
         self.side_effects_helper['UnlockSession']['callerHasLock'] = True
 
+        # For _try_to_get_channel_names()
```

Comment:
there is no function by this name...

But I'm curious why you added this to `nidigital`'s test but not the other modules. If anything this should be in `nifake` no?

Reply:
Thanks for spotting this, this function had been removed in my previous commit. I've updated the comment accordingly in my latest commit.

I ran the unit tests and found that only `nidigital`'s test failed, so I only updated it. Apparently, `nifake` has no `channel_count` property, so its `_all_channels_in_session` property is actually not populated.